src/v/cluster/tests/topic_configuration_compat_test.cc

```diff
@@ -71,6 +74,36 @@ struct topic_configuration_assignment {
     std::vector<cluster::partition_assignment> assignments;
 };
 
+struct incremental_topic_updates {
```

Comment:
It seems there are similar structs defined in `serialization_rt_test.cc`. I wonder what is the canonical location.